derive/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "fueros_derive"
+description = "Derive macros for Fueros"
+version = "0.1.0"
+authors = ["You <you@example.com>"]
```

Comment:
Can you include authors/delete the attribute otherwise?

Reply:
yes master